### PR TITLE
Add shutdown hook

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

**Comment by hepin1989** [comment]
We will need a terminate too

**Comment by regiskuckaertz** [comment]
That's what I thought, yeah. Just a quick fix as I needed it in some code and thought others might too.

**Comment by regiskuckaertz** [comment]
> Or maybe that could be a separate method: awaitIdle

That sounds great, I'll look into it this week-end 👍

File: core/jvm/src/main/scala/scalaz/zio/RTS.scala [hunk 1]
```diff
@@ -14,4 +14,6 @@ trait RTS {
 
   final def unsafeRunAsync[E, A](io: IO[E, A])(k: ExitResult[E, A] => Unit): Unit =
     env.unsafeRunAsync(io, k)
+
+  final def shutdown(): Unit = env.shutdown()
```
Comment (hepin1989) [inline core/jvm/src/main/scala/scalaz/zio/RTS.scala]: should add a method for` awaitTermination ` too.

**Pull request merged**